tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java

```diff
@@ -13,10 +16,21 @@
 @Service
 public class TesTaskServiceImpl implements TesTaskService {
     private final CloudPipelineAPIClient cloudPipelineAPIClient;
+    private final TaskMapper taskMapper;
 
     @Autowired
-    public TesTaskServiceImpl(CloudPipelineAPIClient cloudPipelineAPIClient) {
+    public TesTaskServiceImpl(CloudPipelineAPIClient cloudPipelineAPIClient, TaskMapper taskMapper) {
         this.cloudPipelineAPIClient = cloudPipelineAPIClient;
+        this.taskMapper = taskMapper;
+    }
+
+    @Override
+    public TesCreateTaskResponse submitTesTask(TesTask body) {
+        TesCreateTaskResponse tesCreateTaskResponse = new TesCreateTaskResponse();
+        PipelineRun pipelineRun = cloudPipelineAPIClient.runPipeline(taskMapper.mapToPipelineStart(body));
+        Assert.notNull(pipelineRun.getId(), "INVALID PIPELINE ID");
```

Comment:
See `MessageHelper` class from api module and how it is used in the project, try to realize this approach here and in other places (error throwing, logs, etc)

Reply:
updated